app/controllers/projects_controller.rb

```diff
@@ -100,10 +99,20 @@ def set_properties
   end
 
   def set_aggregation
-    a = Event::AGGREGATIONS.key?(params[:agg]) ? Event::AGGREGATIONS[params[:agg]] : "day"
-    @day_not_allowed = time_period > 6
-    a = "week" if a == "day" && @day_not_allowed
-    @aggregation = CGI.escapeHTML(a)
+    # order array such that .first gives default value
+   @possible_aggregations = case time_period
+                            when 366...5000
```

Comment:
You can use `Float::INFINITY` to say "greater than". So: `when 366..Float::INFINITY`